openmetadata-ui/src/main/resources/ui/src/components/AlertBar/alert-bar.style.less

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+.alert-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+.alert-content {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+}
+
+.cross-icon {
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+}
+
+.general {
```

Comment:
If we are doing like this then it can conflict with other places so can we instead make it more specific to alert container only?

Reply:
updated